### DISCUSSION TITLE
Any update on bringing in TranslateExtension into Maui CommunityToolkit

### DISCUSSION THREAD

**Discussion by egvijayanand** [CommunityToolkit/Maui#578] [General]
TranslateExtension, available as part of Xamarin CommunityToolkit, is a very helpful extension for those who write multi-lingual Apps with XAML.

So, consider bringing that to Maui CommunityToolkit.

**Comment by VladislavAntonyuk** [comment]
It is moved to https://github.com/CommunityToolkit/dotnet/issues/312
@Sergio0694 could you please refine the issue if it is ready to create a PR?

**Comment by egvijayanand** [comment]
Glad to hear that some progress has been made on this.

But after going through the conversations, I don't see finalization on the implementation part. Is it still under discussion?

**Comment by CliffAgius** [comment]
Closed as answered
